### DISCUSSION TITLE
Error connecting to MSSQL Named Instance

### DISCUSSION THREAD

**Discussion by jreilly-lukava** [prisma/prisma#6874] [Q&A] [answered]
If I tried to connect to a named instance with a URL like `sqlserver://goblin\dev;database=stats;user=statsuser;password=statspassword;trustServerCertificate=true;schema=stats` I receive an error message `Error querying the database: Conversion error: SQL browser timeout during resolving instance dev`.

However if I look I look up the port number for that named instance and use it instead everything works as expect.
`sqlserver://goblin:57155;database=stats;user=statsuser;password=statspassword;trustServerCertificate=true;schema=stats`

Any suggestions?  Is this just a bug in the preview feature?

**Comment by janpio** [comment, marked as answer]
Sounds similar to https://github.com/prisma/prisma/issues/6845

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
